src/bundles/config.js

```diff
@@ -13,6 +17,15 @@ const bundle = createAsyncResourceBundle({
       conf = JSON.stringify(rawConf, null, '\t')
     }
 
+    // More info: https://github.com/ipfs-shipyard/ipfs-webui/issues/1490#issuecomment-671633602
+    const config = JSON.parse(conf)
+    const url = getURLFromAddress('Gateway', config) || DEFAULT_URI
+
+    if (!await checkIfGatewayUrlIsAccessible(url)) {
+      config.Addresses.AvailableGateway = DEFAULT_MULTI_ADDR
+      return JSON.stringify(config)
```

Comment:
nit: without below, it will put everything in one line, making JSON editor useless

```suggestion
      return JSON.stringify(config, null, '\t'))
```

Reply:
Removed this, but thanks for the heads up!